src/main.py

```diff
@@ -300,6 +299,7 @@ def run(self):
                                     sub.sticky()
                                     print "Submission stickied..."
                             break
+                        else: time.sleep(120)

```

Comment:
Could you also drop the sleep to 60 seconds?

via: https://github.com/reddit/reddit/wiki/API we can hit it 60 times a minute and currently we're hitting it once every two minutes. Might as well make it update quicker since we can.